PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,3 +31,6 @@ Are all of the CI checks passing?
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing tests pass with my changes
+- [ ] Is the China region affected?
+  - [ ] Are all DB schema changes synchronised?
+  - [ ] Does code handle edge cases correctly?
```

Comment:
I think for this, perhaps changing the wording to something like "Is the code correctly handling the setup in the China region?"

Reply:
Good point, that's more precise!

Reply:
Done!